open_event/static/js/admin/event/new.js

```diff
@@ -0,0 +1,81 @@
+$( document ).ready(function() {
+    var counter = 0;
+    $('.add-session-type').click(function (){
+        var row = "<div class='col-sm-12 row-session-type'>" +
+                    "<div class='col-sm-3'>" +
+                        "<input type='text' class='form-control' name='session_type[name]' placeholder='Name'>" +
+                    "</div>" +
+                    "<div class='col-sm-3 input-group'>" +
+                        "<input type='text' class='form-control' name='session_type[length]' placeholder='Length'>" +
+                        "<span class='input-group-btn'>" +
+                            "<button type='button' class='btn btn-danger remove-session-type'>-</button>" +
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2678420059/issues/source?bid=3233864&fileBranchId=3324174#l11)